input/includes/resourcereferencesummary.md

```diff
@@ -0,0 +1,19 @@
+|Extension|[AU Core Sex Assigned At Birth](StructureDefinition-au-core-rsg-sexassignedab.html)|-|
+|Immunization|[AU Core Immunization](StructureDefinition-au-core-immunization.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|
+|Location|[AU Core Location](StructureDefinition-au-core-location.html)|[AU Core Organization](StructureDefinition-au-core-organization.html)|
+|Medication|[Medication](https://hl7.org/fhir/R4/medication.html), [AU Core Medication](StructureDefinition-au-core-medication.html)|-|
+|MedicationRequest|[AU Core MedicationRequest](StructureDefinition-au-core-medicationrequest.html)|[AU Core Condition](StructureDefinition-au-core-condition.html), [AU Core Encounter](StructureDefinition-au-core-encounter.html), [Medication](https://hl7.org/fhir/R4/medication.html), [AU Core Medication](StructureDefinition-au-core-medication.html), [Observation](http://hl7.org/fhir/R4/observation.html), [AU Core Organization](StructureDefinition-au-core-organization.html), [AU Core Patient](StructureDefinition-au-core-patient.html), [AU Core Practitioner](StructureDefinition-au-core-practitioner.html), [AU Core PractitionerRole](StructureDefinition-au-core-practitionerrole.html), [AU Base Related Person](http://hl7.org.au/fhir/StructureDefinition/au-relatedperson)|
+|Observation|[AU Core Blood Pressure](StructureDefinition-au-core-bloodpressure.html), [AU Core Body Height](StructureDefinition-au-core-bodyheight.html), [AU Core Body Temperature](StructureDefinition-au-core-bodytemp.html), [AU Core Body Weight](StructureDefinition-au-core-bodyweight.html), [AU Core Diagnostic Result Observation](StructureDefinition-au-core-diagnosticresult.html), [AU Core Heart Rate](StructureDefinition-au-core-heartrate.html), [Observation](http://hl7.org/fhir/R4/observation.html), [AU Core Pathology Result Observation](StructureDefinition-au-core-diagnosticresult-path.html), [AU Core Respiration Rate](StructureDefinition-au-core-resprate.html), [AU Core Smoking Status](StructureDefinition-au-core-smokingstatus.html), [AU Core Waist Circumference](StructureDefinition-au-core-waistcircum.html)|[AU Core Diagnostic Result Observation](StructureDefinition-au-core-diagnosticresult.html), [AU Core Organization](StructureDefinition-au-core-organization.html), [AU Core Pathology Result Observation](StructureDefinition-au-core-diagnosticresult-path.html), [AU Core Patient](StructureDefinition-au-core-patient.html), [AU Core Practitioner](StructureDefinition-au-core-practitioner.html), [AU Core PractitionerRole](StructureDefinition-au-core-practitionerrole.html), [AU Base Related Person](http://hl7.org.au/fhir/StructureDefinition/au-relatedperson), [AU Base Specimen](http://hl7.org.au/fhir/StructureDefinition/au-specimen)|
```

Comment:
Please break down by profile - the table should display the supported expectations for each specific profile.

Reply:
Split profiles out to have individual row.